tests/Unit/Evolution/DiscontinuousGalerkin/Test_Filtering.cpp

```diff
@@ -191,15 +238,21 @@ void test_exponential_filter_creation() noexcept {
 SPECTRE_TEST_CASE("Unit.Evolution.dG.ExponentialFilter", "[Unit][Evolution]") {
   // Can't do a loop over different alpha and half_power values because matrices
   // are cached in the action.
-  const double alpha = 20.0;
+  const double alpha = 10.0;
```

Comment:
10 is easier for debugging because the values differ in a more significant digit than 20. It made it easier to figure out where bugs were in the code :)